game_eggs/steamcmd_servers/night_of_the_dead/Night of the dead.json

```diff
@@ -0,0 +1,31 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "exported_at": "2021-07-12T22:41:37+02:00",
+    "name": "NOTD",
+    "author": "info@prospectbyte.nl",
+    "description": "Server egg for Night of the Dead",
+    "features": null,
+    "images": [
+        "quay.io\/parkervcp\/pterodactyl-images:debian_wine-5"
+    ],
+    "file_denylist": [],
+    "startup": "cp \/home\/container\/notd\/ServerSettings.ini \/home\/container\/notd\/LF\/Saved\/Config\/ServerSettings.ini;\r\ncp \/home\/container\/notd\/steamclient64.dll \/home\/container\/notd\/LF\/Binaries\/Win64\/steamclient64.dll;\r\ncp \/home\/container\/notd\/tier0_s64.dll \/home\/container\/notd\/LF\/Binaries\/Win64\/tier0_s64.dll;\r\ncp \/home\/container\/notd\/vstdlib_s64.dll \/home\/container\/notd\/LF\/Binaries\/Win64\/tier0_s64.dll;\r\ncd \/home\/container\/notd\/; \r\nwine \/home\/container\/notd\/LF\/Binaries\/Win64\/LFServer.exe ?listen -log -Port=7777 -QueryPort=27015 -CRASHREPORTS",
+    "config": {
```

Comment:
You should add variables for the game settings in the ServerSettings.ini and use the config parameter to update them when the game starts.

Reply:
Will do that when it actually runs.